Mathlib/Topology/Algebra/Ring/Basic.lean

```diff
@@ -254,6 +254,12 @@ theorem TopologicalRing.of_nhds_zero
   have := TopologicalAddGroup.of_comm_of_nhds_zero hadd hneg hleft
   TopologicalRing.of_addGroup_of_nhds_zero hmul hmul_left hmul_right
 
+theorem TopologicalRing.isInducing_iff_nhds_zero {G H : Type*} [Ring G] [TopologicalSpace G]
+    [Ring H] [TopologicalRing G] [TopologicalSpace H] [TopologicalRing H] (f : G →+* H) :
```

Comment:
Could you instead generalize [TopologicalGroup.isInducing_iff_nhds_one](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Group/Basic.html#TopologicalGroup.isInducing_iff_nhds_one) to `MonoidHomClass`, so that it applies directly to ring homomorphisms? Thanks!